website/content/docs/developers/compiling/compiling.md

```diff
@@ -24,11 +24,10 @@ You must have the following installed to compile Heron:
 * [Bazel](http://bazel.io/docs/install.html) = {{% bazelVersion %}}. Later
   versions might work but have not been tested. See [Installing Bazel]({{< ref
   "#installing-bazel" >}}) below.
-* [Java
-  8](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
-  is required by Bazel but Heron does not use Java 8 features; Heron
+* [Java 8](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+  is required by Bazel and Heron;
   [topologies](../../../concepts/topologies) can be written in Java 7 or above
-  and all Heron jars are compatible with Java 7.
+  , all Heron jars are required to run with Java 8.
```

Comment:
Instead of "all Heron jars are required to run with Java 8" how about "but Heron topologies are required to run with a Java 8 JRE"

/cc @lucperkins, @kramasamy for comment